frontend/src/components/DataTable/DataTable.jsx

```diff
@@ -191,6 +192,7 @@ export default function DataTable({ config, extra = [] }) {
             onChange={filterTable}
             placeholder={translate('search')}
             allowClear
+            suffix={<SearchOutlined />}
```

Comment:
I think this is out of the PR topic. I quote from contribution guidelines "Do not include changes that are not related to the issue at hand."
You can read the contribution guidelines here https://github.com/idurar/idurar-erp-crm/blob/master/CONTRIBUTING.md#how-to-contribute

Reply:
Yes, this actually got commited by mistake